scripts - Restore packages.bat

```diff
@@ -1,4 +0,0 @@
-for /f %%a in ('dir /b src\*.sln') do call tools\nuget\nuget.exe restore src\%%a -PackagesDirectory .\lib
```

Comment:
I know msbuild can take care of this, but it's... so... slow... That's why we want to have a batch file to directly restore the packages without having to open vs.